### ISSUE TITLE
Optenv support in rocon_launch

### ISSUE THREAD

**Issue by jihoonl** [robotics-in-concert/rocon_tools#28]
Does `rocon_launch` support optenv argument?

**Comment by stonier** [comment]
I dont think so. I wanted to use this as well last week when writing the distributable tutorials but didnt get time to check or do.  Lets leave this issue open for it.
- [x] confirm that it really isnt implemented
- [ ] implement it

**Comment by jihoonl** [comment]
https://github.com/robotics-in-concert/rocon_tools/blob/indigo/rocon_launch/src/rocon_launch/utils.py#L38-L84 shows that `optenv` is not implemented.

**Comment by jihoonl** [comment]
But I wonder whether we still need this feature.

**Comment by stonier** [comment]
Certainly be nice to have. Leave it open until we have a need to implement it.
